slugger.cabal

```diff
@@ -0,0 +1,63 @@
+cabal-version:        3.0
+name:                 slugger
+version:              0.1.0.0
+synopsis:             Clean URI slugs for Haskell
+description:          Clean URI slugs for Haskell
+homepage:             https://github.com/rpearce/slugger
+bug-reports:          https://github.com/rpearce/slugger/issues
+license:              BSD-3-Clause
+license-file:         LICENSE
+author:               Robert W. Pearce
+maintainer:           me@robertwpearce.com
+copyright:            2021 Robert W. Pearce
+category:             Data, Text, Web
+extra-source-files:   CHANGELOG.md
+                    , README.md
+
+common lang
+    default-language: Haskell2010
+
+common deps
+    build-depends:
+        base ^>= 4.13
+      , text ^>= 1.2
+      , text-icu ^>= 0.7
+
+common self-dep
+    build-depends: slugger
+
+common flags
+    ghc-options:
+        -Wall
+        -Wcompat
+        -Werror
+        -Widentities
+        -Wincomplete-record-updates
+        -Wincomplete-uni-patterns
+        -Wpartial-fields
+        -Wredundant-constraints
+
+library
+    import:           lang, flags, deps
+    exposed-modules:  Data.String.Slugger
+                      Data.Text.Slugger
+    hs-source-dirs:   lib
+    --other-modules:
```

Comment:
Typo?

Reply:
Totally. I think that was supposed to be commented out, but I can remove